haskell-src/Concordium/Types/Execution.hs

```diff
@@ -828,6 +849,14 @@ instance S.Serialize Event where
               TransferMemo {..} ->
                 S.putWord8 21 <>
                 S.put tmMemo
+              Interrupted {..} ->
+                S.putWord8 22 <>
+                S.put iAddress <>
+                putListOf S.put iEvents
```

Comment:
We could use fewer than 8 bytes for the length here as we will never have that many events.

Reply:
I use 4 bytes now to be consistent with the remaining lists of events (even though 4 bytes is also too much, it must be that for backwards compatibility).